cluster/gce/util.sh

```diff
@@ -85,6 +95,11 @@ function join_csv {
   local IFS=','; echo "$*";
 }
 
+# This function returns the first string before the comma
+function split_csv {
+  echo "$*" | cut -d',' -f1
```

Comment:
It seems like, split_csv should be the inverse of join_csv. Maybe csv_first_element is a better name.